Updating Alert Creation Docs (#27787)
dagster-io/dagster: docs/docs/dagster-plus/features/alerts/creating-alerts.md

<Tabs groupId="ui_or_cli">
  <TabItem value='ui' label='In the UI'>
    1. In the Dagster UI, click **Deployment**.
2. Click the **Alert policies** tab.
3. Click **Create alert policy**.
4. Select **Run** from the menu and click **Continue**.

> 5. Choose run targets and select "Run exceeding a specified number of hours" filling in the number of hours and click **Continue**.

Reviewer: ```suggestion
5. Choose run targets, select "Run exceeding a specified number of hours", enter the number of hours, and click **Continue**.
```

<Tabs groupId="ui_or_cli">
  <TabItem value='ui' label='In the UI'>
    1. In the Dagster UI, click **Deployment**.
2. Click the **Alert policies** tab.
3. Click **Create alert policy**.
4. Select **Asset** from the menu and click **Continue**.

> 5. Choose asset targets and select the relevant asset check events and click **Continue**.

Reviewer: ```suggestion
5. Choose asset targets, select the relevant asset check events, and click **Continue**.
```

<Tabs groupId="ui_or_cli">
  <TabItem value='ui' label='In the UI'>
    1. In the Dagster UI, click **Deployment**.
2. Click the **Alert policies** tab.
3. Click **Create alert policy**.
4. Select **Automation** from the menu and click **Continue**.

> 5. Select the tick types to be alerted on. Choose if you only want to be alerted after N consecutive failures and click **Continue**.

Reviewer: ```suggestion
5. Select the tick types to be alerted on. Choose if you only want to be alerted after `n` consecutive failures and click **Continue**.
```
